### PR TITLE
Zenko 3362 enhance ux account

### PR DESCRIPTION
**Pull request by theocerutti**
image
image

### CODE REVIEW

**Review by Cuervino — CHANGES_REQUESTED** [review]
- ZENKO-3183 Account details title to add

Could you please align the new title with the content of the table below? It looks like there is at least 1 px discrepancy.
image

- ZENKO-3184 'Created On' Column format to adjust on Account list
Thanks, it's good!

File: src/react/account/AccountList.jsx [hunk 1]
```diff
@@ -27,7 +27,8 @@ const columns = [
     {
         Header: 'Created on',
         accessor: 'createDate',
-        Cell: ({ value }) => { return formatDate(new Date(value));},
+        headerStyle: { textAlign: 'right' },
+        Cell: ({ value }) => <div style={{ textAlign: 'right' }}>{formatSimpleDate(new Date(value))}</div>,
```
Comment (nicolas2bert) [inline src/react/account/AccountList.jsx]: We might want to create a `ui-elements/Utility.js` file and import styled component `textAlign: 'right'` from there for reusability.

File: src/react/ui-elements/TableKeyValue.jsx [hunk 2]
```diff
@@ -42,6 +42,7 @@ export const ExtraCell = styled.td`
 `;
 
 const Table = styled.table`
+    border-spacing: 0;
```
Comment (nicolas2bert) [inline src/react/ui-elements/TableKeyValue.jsx]: unity?

File: src/react/account/AccountList.jsx (hunk 1 above)
Comment (theocerutti) [reply src/react/account/AccountList.jsx]: done

File: src/react/ui-elements/TableKeyValue.jsx (hunk 2 above)
Comment (theocerutti) [reply src/react/ui-elements/TableKeyValue.jsx]: done

**Review by Cuervino — APPROVED** [review]
LGTM!

**Review by terrydervaux — APPROVED** [review]
LGTM

**Review by nicolas2bert — APPROVED** [review]

**Comment by theocerutti** [comment]
@bert-e approve

**Pull request merged**